File: redsheet-portal/src/app/database-models/v-their-team-member.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vTheirTeamMember {
    constructor(vTheirTeamMember?: vTheirTeamMember )  {
        let properties = [ "personalityId", "personalityKeyText", "position", "projectNegotiationId", "theirTeamMemberId", "theirTeamMemberName",];
        for (let property of properties)
            if (vTheirTeamMember && vTheirTeamMember[property])
                this[property] = vTheirTeamMember[property];
    }
 
	personalityId : number =   0 ;
 
    @maxLength(10)
	personalityKeyText : string =   undefined;
 
    @required()
    @maxLength(500)
	position : string =   undefined;
 
    @range(1,2147483647)
	projectNegotiationId : number =   undefined;
 
    @range(1,2147483647)
	theirTeamMemberId : number =   undefined;
 
    @required()
    @maxLength(50)
	theirTeamMemberName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vTheirTeamMember.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vTheirTeamMembers", Schema = "dbo")]
	[Serializable]
	public partial class vTheirTeamMember  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int PersonalityId { get; set; } 
 
	[MaxLength(10)]
	public string PersonalityKeyText { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string Position { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectNegotiationId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int TheirTeamMemberId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string TheirTeamMemberName { get; set; } 

	public vTheirTeamMember()
	{
    }

    }
}

